src/components/general/Slider/index.tsx

```diff
@@ -112,7 +117,7 @@ const Slider: React.FC<SliderProps> = ({ value, markers, disabled, onChange }) =
                 }}
             />
             <button
-                className={styles.handle}
+                className={handleClassNames}

```

Comment:
Wouldn't it be better to not render the handle at all?

Reply:
Good point, handle will now not render on hide